src/less/main.less

```diff
@@ -377,7 +377,9 @@
   border-radius: @radius-default;
   display: none;
   margin: 0 auto;
-  max-height: 500px;
+  // NEXT_MAJOR_VERSION drop this when IE support is no longer required
+  max-height: 600px; // IE11 doesn't support fit-content, so we need a backup value here
+  max-height: fit-content;
```

Comment:
We tried to find context around why this was originally set to 500px but couldn't find it; it seemed almost arbitrary. We would like to know if there are some ramifications to changing this to `fit-content` that we might be missing.

Reply:
This note _feels_ right.